CHANGELOG.md

```diff
@@ -41,6 +41,7 @@ Additional documentation and release notes are available at [Multiplayer Documen
   * Removed DynamicNavObstacle - an unused class (#732)
   * Merged networked data classes into their Server counterparts. An example of that change is the contents of NetworkCharacterState getting moved into ServerCharacter, contents of NetworkDoorState getting moved into SwitchedDoor etc. (#732)
 * Engine version bump to 2021.3.10f1 (#740)
+* Removed unnecessary FindObjectOfType usage inside of ClientCharSelectState (#754)
```

Comment:
Since this is going post release, adding a note to add this release note in another section afterward.

Reply:
Done.